migrations/sql/V2019.08.01.13.17__remove_region_suffix_from_mine_region.sql

```diff
@@ -0,0 +1,5 @@
+UPDATE mine_region_code set description = 'South West' where description = 'South West Region';
```

Comment:
It makes sense in this case because we're updating description specifically from one to another, but it's generally best to use a PK